services/catalog/src/simcore_service_catalog/api/rest/_services.py

```diff
@@ -237,17 +247,19 @@ async def get_service(
                 status_code=status.HTTP_403_FORBIDDEN,
                 detail="You have insufficient rights to access the service",
             )
-    # access is allowed, override some of the values with what is in the db
-    service = service.copy(
-        update=service_in_db.dict(exclude_unset=True, exclude={"owner"})
-    )
+
     # the owner shall be converted to an email address
     if service_in_db.owner:
-        service.owner = await groups_repository.get_user_email_from_gid(
+        service_data["owner"] = await groups_repository.get_user_email_from_gid(
             service_in_db.owner
         )
 
-    return service
+    # access is allowed, override some of the values with what is in the db
+    service_in_registry = service_in_registry.copy(
+        update=service_in_db.dict(exclude_unset=True, exclude={"owner"})
+    )
+    service_data.update(service_in_registry.dict(exclude_unset=True, by_alias=True))
+    return service_data
```

Comment:
minor: not sure why you return a dictionary here. it looks less secure, and more prone to being forgotten when one rename a variable.

Reply:
@sanderegg it is mostly a style decision to avoid redundancy. 

NOTE that fastapi ALWAYS validate this returned value with the class passed in the decorator parameter `response_model`. In this case `response_model=ServiceGet` and therefore if I return an instance of `ServiceGet` probably it will not validate it again but I found that it was kind of duplicated.